internal/hcs/schema2/app_container_launch_type.go

```diff
@@ -0,0 +1,24 @@
+// Autogenerated code; DO NOT EDIT.
+
+// Schema retrieved from branch 'rs_base2_hyp' and build '27645.1002.240617-2119'.
```

Comment:
we should remove the branch/build manually I think.

Reply:
Ok, done.